src/Texture.js

```diff
@@ -467,6 +467,11 @@ x3dom.Texture.prototype.updateTexture = function ()
             tex._video.play()
                 .then( function fulfilled ()
                 {
+                    if ( tex._intervalID )
```

Comment:
When does that happen ? Probably startVideo() should not be called in the first place when already updating.

Reply:
If _intervalID is a non-zero number, it means that there is already a running interval, and it must be closed before setting a new one.

Reply:
Yes. I feel the guard may not needed and would rather hide any underlying problem. Could you add a logWarning message ?

Reply:
If this guard is removed, it will be difficult to troubleshoot errors caused by repeated calls to startVideo().